src/server/scripts/Northrend/Ulduar/Ulduar/boss_xt002.cpp

```diff
@@ -230,7 +230,7 @@ class boss_xt002 : public CreatureScript
 
                 events.ScheduleEvent(EVENT_ENRAGE, TIMER_ENRAGE);
                 events.ScheduleEvent(EVENT_GRAVITY_BOMB, TIMER_GRAVITY_BOMB);
-                events.ScheduleEvent(EVENT_SEARING_LIGHT, TIMER_SEARING_LIGHT);
+                events.ScheduleEvent(EVENT_SEARING_LIGHT, TIMER_SEARING_LIGHT / 2);
```

Comment:
Since TIMER_SEARING_LIGHT is used twice as /2 and just once with the normal value, why not set its value to 10000 and then use TIMER_SEARING_LIGHT * 2 in just one of the schedules.